http4s/src/main/resources/reference.conf

```diff
@@ -77,6 +77,11 @@
     port = 443
   }
 
+  networking {
+    maxConnections = 1024
```

Comment:
In prod we use `8192` by default

Reply:
Yea, I wanted to keep it the same as the defaults in akka until we properly calculate what makes the most sense for our processing patterns. Good spot regardless.